src/main/java/mate/academy/service/AccommodationServiceImpl.java

```diff
@@ -0,0 +1,82 @@
+package mate.academy.service;
```

Comment:
replace all implementation to impl 
```suggestion
package mate.academy.service.impl;
```